### DISCUSSION TITLE
Unsure of what is happening

### DISCUSSION THREAD

**Discussion by toastie-Sib** [ramaureirac/godot-tactical-rpg#19] [General]
When I go through the project and open up the files to take a look at how everything in working, I eventually start getting parse errors upon opening certain files. I am not sure why this is happening. The project seems to be working fine regardless, but I am not sure if the best course of action is to just close the issued scripts to make the errors go away. 
![image](https://github.com/user-attachments/assets/cde93223-22ef-4c0e-9648-9feff45e9754)

**Comment by dukemagus** [comment]
Try to reload the project

**Reply by toastie-Sib** [reply]
Reloading the project ended up looking like the same problem. All the errors came up again. I see no issues while playtesting though, so I'm not sure if it is because of me opening all the files I could try to take a look at? That is when the errors would pop up, and I believe that closing scripts makes them go away. 
![image](https://github.com/user-attachments/assets/59ee056d-3776-4f41-94e7-b8c1d80d995d)

**Comment by max-koro** [comment]
I had a similar issue. Fixed it by closing all scenes and all script windows before reloading the project (Project → Reload Current Project). However, it reappears whenever the new pawn scene is instantiated. It looks like Godot is loading twice the same class for some reason.

I'll look into it and submit a fix if I isolate the issue

**Comment by max-koro** [comment]
@toastie-Sib [There you go](https://github.com/ramaureirac/godot-tactical-rpg/pull/20) 😄

**Comment by toastie-Sib** [comment]
Perfect! Thank you guys for all your help!!
